reporting-page/frontend/test/privacy-dashboard.test.js

```diff
@@ -120,14 +119,12 @@ describe('Privacy dashboard page', function() {
       if (index == 1) mockRiskCounters.lastHighRisk = 0;
       if (index == 2) mockRiskCounters.lastMediumRisk = 0;
       if (index == 3) mockRiskCounters.lastLowRisk = 0;
-      if (index == 4) mockRiskCounters.lastInfoRisk = 0;
+      if (index == 4) mockRiskCounters.noRiskColor = 0;
```

Comment:
Is this correct for the test? Since this should test what the status is when the risk counters hit a certain threshold.